README.md

```diff
@@ -166,3 +166,51 @@ amount     = 1000
 
 Justifi::Payment.create_refund( amount: 1000, reason: reason, payment_id: payment_id )
 ```
+
+## Listing Resources
+
+All top-level API resources have support for bulk fetches via `array` API methods.
+JustiFi uses cursor based pagination which supports `limit`, `before_cursor` and `after_cursor`.
+Each response will have a `page_info` object that contains the `has_next` and `has_previous` fields,
+which tells you if there are more items before or after the current page.
+The `page_info` object also includes `start_cursor` and `end_cursor` values which can be used in 
+conjuction with `before_cursor` and `after_cursor` to retrieve items from the api one page at a time.
+
+### List PaymentMethods
+
+```ruby
+require 'justifi'
+
+# gem setup...
+
+pms = Justifi::PaymentMethod.list
```

Comment:
I removed PaymentMethod because the API  it's not ready yet but put a link to our developer portal.

I personally find it easier to keep track, specially right now that we're still changing the docs there.